src/suit/js/SUIT.js

```diff
@@ -81,46 +80,14 @@ AND ( t_min <= 61000 AND 60900 <= t_max )
 AND ( 600e-9 BETWEEN em_min AND em_max )`
             },
         ]
-    });
+    };
 
-let tapServices= getTAPServices( ['IRSA', 'Gaia', 'CADC', 'MAST Images',
-    'GAVO', 'HSA', 'NED', 'NASA Exoplanet Archive']);
+const tapServices=  [
+    lsstEntry,
+    ...getTAPServices( ['IRSA', 'Gaia', 'CADC', 'MAST Images', 'GAVO', 'HSA', 'NED', 'NASA Exoplanet Archive'])
+];
 
 
-const TAP_PATH= 'api/tap';
-
-/**
```

Comment:
What's going on with the removal of all this logic?

Reply:
It is really just a hack and is it is not longer needed with out better deployment methods.